internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/endToEnd/TestActiveActiveIngestionWithReadWriteLeaderCompactionTuningAndParallelProcessing.java

```diff
@@ -1,11 +1,18 @@
 package com.linkedin.venice.endToEnd;
 
 /**
- * Integration test to verify active/active replication when applying read-write leader compaction optimization.
+ * Integration test to verify active/active replication when applying read-write leader compaction optimization
+ * and parallel processing of AA/WC workload.
  */
-public class TestActiveActiveIngestionWithReadWriteLeaderCompactionTuning extends TestActiveActiveIngestion {
+public class TestActiveActiveIngestionWithReadWriteLeaderCompactionTuningAndParallelProcessing
+    extends TestActiveActiveIngestion {
   @Override
   protected boolean isLevel0CompactionTuningForReadWriteLeaderEnabled() {
     return true;
   }
```

Comment:
This change is pretty important, is it possible that we add a new test that sends a bunch of updates with same keys? I am not sure if this scenario is covered in existing tests.

Reply:
`TestChangelogConsumer#testAAIngestionWithStoreView` has this pattern, and that is the place I discovered the race condition with changelog view.